tests/series/drop_nulls_test.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from typing import Any
+
+import narwhals as nw
+from tests.utils import compare_dicts
+
+
+def test_drop_nulls(constructor: Any) -> None:
+    data = {
+        "A": [1, 2, None, 4],
+        "B": [5, 6, 7, 8],
+        "C": [None, None, None, None],
+        "D": [9, 10, 11, 12],
+    }
+
+    df = nw.from_native(
+        constructor(data), strict=False, eager_only=True, allow_series=True
+    )
+
+    result_a = df.select(nw.col("A").drop_nulls())
+    result_b = df.select(nw.col("B").drop_nulls())
+    result_c = df.select(nw.col("C").drop_nulls())
+    result_d = df.select(nw.col("D").drop_nulls())
```

Comment:
can you do all of these in the same select statement? so
```
result = df.select(
    nw.col('a').drop_nulls(),
    nw.col('b').drop_nulls(),
)
```
etc...

and then just make a single dict for `expected`

Reply:
as per https://github.com/narwhals-dev/narwhals/pull/542#pullrequestreview-2182111442, this is OK, let's keep it like this, sorry for having lead you in the wrong direction